barretenberg/cpp/src/barretenberg/vm/avm/generated/verifier.cpp

```diff
@@ -11,6 +11,7 @@ namespace bb {
 
 AvmVerifier::AvmVerifier(std::shared_ptr<Flavor::VerificationKey> verifier_key)
     : key(verifier_key)
+    , pcs_verification_key(std::make_shared<VerifierCommitmentKey>())
```

Comment:
Ah, so it's used here. Where do we create a verifier without a valid VK, and the question is: should we?